cpg-analysis/src/test/kotlin/de/fraunhofer/aisec/cpg/testcases/FlowQueriesTest.kt

```diff
@@ -61,7 +62,7 @@ class FlowQueriesTest {
                     .defaultPasses()
                     .registerPass<ControlDependenceGraphPass>()
                     .registerPass<ProgramDependenceGraphPass>()
-                    .registerLanguage(TestLanguage("."))
```

Comment:
Can we please avoid such changes within a PR having some actual changes wrt. the functionality in the future? It makes it much more time-consuming to review the PR and changes.

Reply:
As discussed, this is unfortunately necessary because we changed the creation point from the config to the manager.